src/main/java/marquez/core/models/JobRunState.java

```diff
@@ -0,0 +1,60 @@
+package marquez.core.models;
+
+import java.sql.Timestamp;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.Data;
+
+@Data
+public class JobRunState {
+  private final UUID guid;
+  private final Timestamp transitionedAt;
+  private final UUID jobRunGuid;
+  private final State state;
+
+  public JobRunState(
+      final UUID guid,
+      final Timestamp transitionedAt,
+      final UUID jobRunGuid,
+      final JobRunState.State state) {
+    this.guid = guid;
+    this.transitionedAt = transitionedAt;
+    this.jobRunGuid = jobRunGuid;
+    this.state = state;
+  }
+
+  public enum State {
+    NEW,
+    RUNNING,
+    COMPLETED,
+    FAILED,
+    ABORTED;
```

Comment:
I think this exists in the other JobRunState.java file - maybe we should factor it out and reference instead of repeating it?

Reply:
I didn't refactor across core models and API models in this PR, as it already had quite a bit of scope. I just removed the dead code from this class. 👍 to refactoring further in a future PR

Reply:
the JobRun API model probably only needs a string for the state (e.g. "NEW", "RUNNING"), not the enum anymore.